src/main/scala/io/qbeast/spark/table/IndexedTable.scala

```diff
@@ -193,16 +195,30 @@ private[table] class IndexedTableImpl(
 
   private def write(data: DataFrame, indexStatus: IndexStatus, append: Boolean): BaseRelation = {
     val revision = indexStatus.revision
-
-    if (exists) {
-      keeper.withWrite(tableID, revision.revisionID) { write =>
-        val announcedSet = write.announcedCubes.map(revision.createCubeId)
+    keeper.withWrite(tableID, revision.revisionID) { write =>
+      var tries = 2
```

Comment:
Should it be a config property? It may make sense to tune it on different scenarios

Reply:
Mm, makes sense. Perhaps something like `spark.qbeast.keeper.numOfTries`?